content/en/contribute/development.md

```diff
@@ -154,9 +154,11 @@ go install
 Hugo relies on [mage](https://github.com/magefile/mage) for some convenient build and test targets. If you don't already have it, get it:
 
 ```
-go get github.com/magefile/mage
+go get github.com/magefile/mage@latest
 ```
 
+This will install mage executable in `$GOPATH` without affecting current project’s module.
```

Comment:
Remove this line. It's not (at least not in general) correct, and it's outside the scope of this change.

Reply:
Updated.